src/App.js

```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Home from "./pages/Home";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Question from "./pages/Question";
 
 function App() {
   return (
```

Comment:
At this point, because you used the Es6 for all of your functions, You should also use ES6 here too.
```suggestion
 const App = () => {
```